Source/IGListAdapter.m

```diff
@@ -759,6 +760,23 @@ - (__kindof UICollectionReusableView *)dequeueReusableSupplementaryViewOfKind:(N
     return [collectionView dequeueReusableSupplementaryViewOfKind:elementKind withReuseIdentifier:identifier forIndexPath:indexPath];
 }
 
+- (__kindof UICollectionReusableView *)dequeueReusableSupplementaryViewOfKind:(NSString *)elementKind
+                                                         forSectionController:(IGListSectionController <IGListSectionType> *)sectionController
+                                                                      nibName:(NSString *)nibName
+                                                                       bundle:(NSBundle *)bundle
+                                                                      atIndex:(NSInteger)index {
+    IGAssertMainThread();
```

Comment:
@rawlinxx no worries! PRs are an iterative process, its good to get something up and we can all work on it together 👨‍👩‍👧‍👦